Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -167,10 +171,23 @@ public override void OnInputChanged(InputEventData<Vector2> eventData)
         {
             if (eventData.SourceId == Controller?.InputSource.SourceId)
             {
-                if (!UseSourcePoseData &&
-                    PoseAction == eventData.MixedRealityInputAction)
+                if (PoseAction == eventData.MixedRealityInputAction && !UseSourcePoseData)
                 {
-                    UpdateMousePosition(eventData.InputData.x, eventData.InputData.y);
+                    Vector3 mouseDeltaRotation = Vector3.zero;
+                    mouseDeltaRotation.x += eventData.InputData.x;
+                    mouseDeltaRotation.y += eventData.InputData.y;
+                    UpdateMouseRotation(mouseDeltaRotation * speed);
+                }
+            }
+        }
+
+        public override void OnInputChanged(InputEventData<MixedRealityPose> eventData)
```

Comment:
>OnInputChanged [](start = 29, length = 14)

ControllerPoseSynchronizer is already doing this for us.

Reply:
the controllerPoseSynchronizer is not calling our method for calculating the rotation - I have to keep this method